src/hooks/screen/useChatScreenHook.ts

```diff
@@ -73,6 +75,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
           userId,
           selectedChannel?.id,
           message,
+          attachments,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` parameter is now being passed to the `ChatSchema.create` method. Make sure that the `create` method in `ChatSchema` is updated to handle this new property.
```diff
+           attachments,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->